test/e2e/tls_test.go

```diff
@@ -65,14 +68,14 @@ func TestAntreaApiserverTLSConfig(t *testing.T) {
 		apiserver     int
 		apiserverStr  string
 	}{
-		{"ControllerApiserver", controllerPodName, controllerContainerName, apis.AntreaControllerAPIPort, "Controller"},
-		{"AgentApiserver", agentPodName, agentContainerName, apis.AntreaAgentAPIPort, "Agent"},
+		{"ControllerApiserver", toolboxPodName, "toolbox", apis.AntreaControllerAPIPort, "Controller"},
+		{"AgentApiserver", toolboxPodName, "toolbox", apis.AntreaAgentAPIPort, "Agent"},
```

Comment:
You can remove podName and containerName fields in the struct since they are the same for both cases.
You can pass  `toolboxPodName` and `toolbox` to data.checkTLS directly.